compiler/src/dotty/tools/dotc/core/tasty/TastyUnpickler.scala

```diff
@@ -49,7 +49,7 @@ class TastyUnpickler(reader: TastyReader) {
       case UTF8 =>
         goto(end)
         termName(bytes, start.index, length)
-      case QUALIFIED | FLATTENED | EXPANDED | EXPANDPREFIX =>
+      case QUALIFIED | FLATTENED | EXPANDED | EXPANDPREFIX | TRAITSETTER =>
```

Comment:
I propose we make an assert instead that triggers when TRAITSETTER is found. It would be good to find out why it arises.